Packs/GoogleThreatIntelligence/Integrations/FeedIoCStream/FeedIoCStream.py

```diff
@@ -0,0 +1,459 @@
+
+    indicator_obj['fields'].update({
+        'creationdate': attributes.get('creation_date'),
+        'admincountry': admin_country.group(1) if admin_country else None,
+        'adminemail': admin_email.group(1) if admin_email else None,
+        'adminname': admin_name.group(1) if admin_name else None,
+        'adminphone': admin_phone.group(1) if admin_phone else None,
+        'registrantcountry': registrant_country.group(1) if registrant_country else None,
+        'registrantemail': registrant_email.group(1) if registrant_email else None,
+        'registrantname': registrant_name.group(1) if registrant_name else None,
+        'registrantphone': registrant_phone.group(1) if registrant_phone else None,
+        'registrarabuseemail': registrar_abuse_email.group(1) if registrar_abuse_email else None,
+        'registrarabusephone': registrar_abuse_phone.group(1) if registrar_abuse_phone else None,
+        'registrarname': attributes.get('registrar'),
+        'firstseenbysource': attributes.get('first_seen_itw_date'),
+        'lastseenbysource': attributes.get('last_seen_itw_date'),
+    })
+
+    return indicator_obj
+
+
+def _add_url_attributes(indicator_obj: dict, attributes: dict) -> dict:
+    """Adds URL attributes."""
+    indicator_obj['fields'].update({
+        'firstseenbysource': attributes.get('first_submission_date'),
+        'lastseenbysource': attributes.get('last_submission_date'),
+    })
+
+    return indicator_obj
+
+
+def _add_ip_attributes(indicator_obj: dict, attributes: dict) -> dict:
+    """Adds IP attributes."""
+    indicator_obj['fields'].update({
+        'countrycode': attributes.get('country'),
+        'firstseenbysource': attributes.get('first_seen_itw_date'),
+        'lastseenbysource': attributes.get('last_seen_itw_date'),
+    })
+
+    return indicator_obj
+
+
+def _add_dedicated_attributes(indicator_obj: dict, attributes: dict) -> dict:
+    """Adds dedicated attributes to indicator object."""
+    if indicator_obj['type'] == FeedIndicatorType.File:
+        return _add_file_attributes(indicator_obj, attributes)
+    if indicator_obj['type'] == FeedIndicatorType.Domain:
+        return _add_domain_attributes(indicator_obj, attributes)
+    if indicator_obj['type'] == FeedIndicatorType.URL:
+        return _add_url_attributes(indicator_obj, attributes)
+    if indicator_obj['type'] == FeedIndicatorType.IP:
+        return _add_ip_attributes(indicator_obj, attributes)
+    raise ValueError(f'Unknown type: {indicator_obj["type"]}. ID: {indicator_obj["id"]}')
+
+
+def _create_indicator(item: dict) -> dict:
+    """Creates indicator object."""
+    indicator_type = _get_indicator_type(item)
+    indicator_id = _get_indicator_id(item)
+
+    attributes = item.get('attributes', {})
+    context_attributes = item.get('context_attributes', {})
+
+    detection_ratio = DetectionRatio(attributes.get('last_analysis_stats', {}))
+
+    indicator_obj = {
+        'type': indicator_type,
```

Comment:
why you didnt added **score** value to the **indicator_obj**?
please see this articles:
https://xsoar.pan.dev/docs/integrations/dbot#score-types
https://xsoar.pan.dev/docs/integrations/dbot#score-types

Reply:
Done!